generics/services/survey.js

```diff
@@ -444,6 +444,54 @@ const userSurveySubmissions = function( token, solutionId ) {
         }
     })
 }
+const getObservationInfo = function (
+    {userToken,stats='false'}
```

Comment:
pass as variable not as object @borkarsaish65

Reply:
done